Cleanup VAST output (#602)
tenzir/vast: CHANGELOG.md

- 🎁 Configuring how much status information gets printed to STDERR previously
  required obscure config settings. From now on, users can simply use
  `--verbosity=<level>,-v <level>`, where `<level>` is one of `quiet`, `error`,
  `warn`, `info`, `debug`, or `trace`. However, `debug` and `trace` are only

>   available on debug builds (otherwise they fall back to `info` output level).

Reviewer: ```suggestion
  available for debug builds (otherwise they fall back to log level `info`).
```
